### PR TITLE
Change db to postgresql

### PR DESCRIPTION
**Pull request by VanyaNovak**
# Checklist:

- [x] Added specs for all new ruby code
- [x] Green rspec run
- [x] Green rubocop
- [x] PR review from teammate

# What?
Was changed db from sqlite3 to postgresql

## Why?
Before we used pg only in production, but now we need to use it locally for simplicity

## How?
`rails db:system:change --to=postgresql`

## Testing?
nope

## Anything Else?
Yep. 

**Instructions for using PostgreSQL on the local machine**

- You must install postgresql to get started https://www.postgresql.org/download/
- Next you need to enter the following command in the terminal `sudo -u postgres createuser --superuser Myname`
P.S. Change `Myname` to your username like `gelvan`@gelvan-Laptop in your terminal)

### CODE REVIEW

**Review by KostyaKobel — CHANGES_REQUESTED** [review]
Add the ENV variables to the database and username variables

File: config/database.yml [hunk 1]
```diff
@@ -1,27 +1,93 @@
...
-    pool: 5
-    timeout: 5000
-    username: postgres
-    password: postgres
-    port: 5432
-    pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 15 } %>
-    url: <%= ENV['DATABASE_URL'] %>
\ No newline at end of file
+  adapter: postgresql
+  encoding: unicode
```
Comment (montashevych) [inline config/database.yml:89]: we don't need repeat this information, you can use <<: *default + url + mb database name.

File: config/database.yml [hunk 2]
```diff
@@ -1,27 +1,93 @@
...
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem 'pg'
+#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
```
Comment (montashevych) [inline config/database.yml:19]: I think we need "timeout" params in default or in production. Have you tried run docker-compose with this params? If my remember is correct, I had some problem with this.

File: config/database.yml (hunk 1 above)
Comment (VanyaNovak) [reply config/database.yml:89]: ok

File: config/database.yml (hunk 2 above)
Comment (VanyaNovak) [reply config/database.yml:19]: Added timeouts

File: config/database.yml (hunk 1 above)
Comment (VanyaNovak) [reply config/database.yml]: Refactored `database.yml `

Comment (VanyaNovak) [reply config/database.yml]: if store database name in default that will be give error

**Review by montashevych — APPROVED** [review]
well done I think)

**Review by Hunter2065 — APPROVED** [review]
I think all is good bro)

**Review by KostyaKobel — APPROVED** [review]
All fine

File: .github/workflows/workflow.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ env:
   POSTGRES_USER: postgres
```
Comment (andrjuha01) [inline .github/workflows/workflow.yml:3]: can we move those to env vars?

File: config/database.yml [hunk 4]
```diff
@@ -1,27 +1,95 @@
...
-    encoding: unicode
-    pool: 5
-    timeout: 5000
-    username: postgres
-    password: postgres
-    port: 5432
-    pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 15 } %>
-    url: <%= ENV['DATABASE_URL'] %>
\ No newline at end of file
+  adapter: postgresql
```
Comment (andrjuha01) [inline config/database.yml:88]: remove adapter and encoding

Comment (VanyaNovak) [reply config/database.yml]: Removed

**Review by andrjuha01 — APPROVED** [review]

**Pull request merged**